guides/puppet-web-apps/html-views/README.md

```diff
@@ -0,0 +1,98 @@
+# HTML Views
+
+This page gives an explaination of partial HTML views and view-models, or, in other words, nested pages.
+
+## Partial HTML View
+
+A Partial HTML View is a valid HTML document that contains at least one `<template>` tag and can be loaded with the [`<imported-template>`](https://github.com/Juicy/imported-template) web component. The `<template>` tag is cached on the client and stamped out as many times as needed.
+
+The requirements and behaviors for partial HTML views originate from the [HTML Imports](http://www.w3.org/TR/html-imports/) and [HTML Template](http://www.w3.org/TR/html-templates/) specifications. The most interesting aspects are:
+
+* The document should be a valid HTML document and can contain anything that the browser allows
+* The document may include dependencies such as stylesheets, HTML imports, scripts, etc.
+* The content outside of the `<template>` tag is requested and executed once per session on partial load time
+* The content inside the `<template>` is stamped out with the `<template>` itself every time the response is bound to a tree in the view-model
+* Every node from the `<template>` will be stamped with an attached model property. Custom elements, template binding frameworks, or JavaScript scripts can use this data to populate the view
+
+## Partial View-Model
```

Comment:
:neckbeard: Removing the special meaning of the word partial. 

Could this section be renamed from "Partial View-Model" to "View-Model Definition"?

Also, the next sentence could begin with "A View-Model Definition is a valid JSON document...".

I am not 100% happy with the above either, but maybe we can get back to that at some other time.

----

At the end of this section we can add a paragraph:

> A "partial" view-model is a view-model that has been nested as part of a bigger view-model. See **Adding the Blending Point**

> A "partial HTML view" together with its corresponding "partial view-model" can simply be referred to as a partial.

Again, I am still not 100% happy with the above, but I think it is an improvement and we can give it a go.